app/javascript/components/components/Footer.js

```diff
@@ -1,18 +1,17 @@
 import React, { Component } from "react"
-import { NavLink } from 'react-router-dom'
 
 class Footer extends Component {
   render() {
     return (
       <>
         <footer>
-          <NavLink className={"about-us"} to={`/aboutus`}>
+          <a className={"about-us"} href={`/aboutus`}>
```

Comment:
I would recommend using NavLink from react-router-dom here as this is navigating internally on this app.

Reply:
Switched back to NavLink, we were experimenting with a tags when we couldn't get not found to show when signed in. We had moved header and footer outside the router.